src/components/OfflineIndicator.tsx

```
@@ -0,0 +1,61 @@
+  return (
+    <div
+      className={cn(
+        'fixed top-4 right-4 z-50 transition-all duration-300 transform',
+        isOnline ? 'translate-y-0 opacity-100' : 'translate-y-0 opacity-100'
+      )}
+    >
+      {!isOnline && (
+        <div className="bg-orange-500 text-white px-4 py-3 rounded-lg shadow-lg flex items-center gap-2 max-w-sm">
+          <WifiOff className="w-5 h-5 flex-shrink-0" />
+          <div>
+            <p className="font-medium">You're offline</p>
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)